### PR TITLE
use err.Error() can lost detail in some case

### PR DESCRIPTION
**Pull request by secmask**

### CODE REVIEW

**Comment by secmask** [comment]
zap/sentry can give more detail if we pass `err` instead of `err.Error()`

File: exchange/huobi.go [hunk 1]
```diff
@@ -764,7 +764,7 @@ func (h *Huobi) WithdrawStatus(
 	}
 	withdraws, err := h.interf.WithdrawHistory(tokens)
 	if err != nil {
-		return "", "", fmt.Errorf("can't get withdraw history from huobi: %s", err.Error())
+		return "", "", fmt.Errorf("can't get withdraw history from huobi: %v", err)
```
Comment (ducquangkstn) [inline exchange/huobi.go]: change to errors.Wrap

File: http/settings_handler.go [hunk 2]
```diff
@@ -407,7 +407,7 @@ func (s *Server) ensureInternalSetting(tokenUpdate common.TokenUpdate) error {
 	token := tokenUpdate.Token
 	if !token.IsETH() { // TokenIndices doesn't contains ETH
 		if uErr := s.blockchain.CheckTokenIndices(ethereum.HexToAddress(token.Address)); uErr != nil {
-			return fmt.Errorf("cannot get token indice from smart contract (%s) ", uErr.Error())
+			return fmt.Errorf("cannot get token indice from smart contract (%+v) ", uErr)
```
Comment (ducquangkstn) [inline http/settings_handler.go]: change to `errors.Wrap` pls

File: http/server.go [hunk 3]
```diff
@@ -105,7 +105,7 @@ func eligible(ups, allowedPerms []Permission) bool {
 func (s *Server) Authenticated(c *gin.Context, requiredParams []string, perms []Permission) (url.Values, bool) {
 	err := c.Request.ParseForm()
 	if err != nil {
-		httputil.ResponseFailure(c, httputil.WithReason(fmt.Sprintf("Malformed request package: %s", err.Error())))
+		httputil.ResponseFailure(c, httputil.WithReason(fmt.Sprintf("Malformed request package: %v", err)))
```
Comment (ducquangkstn) [inline http/server.go]: According to what I have test, `%v` have same effect with `%s`, pls change it to `%+v`

File: http/settings_handler.go (hunk 2 above)
Comment (secmask) [reply http/settings_handler.go]: ok

File: exchange/huobi.go (hunk 1 above)
Comment (secmask) [reply exchange/huobi.go]: ok

File: http/server.go (hunk 3 above)
Comment (secmask) [reply http/server.go]: %s and %v is the same if error is not point to a struct, otherwise it's different.

**Pull request merged**